src/App.css

```diff
@@ -33,6 +32,44 @@
   color: #61dafb;
 }
 
+.Background-logo {
+  background-color: darkred;
+  background-image: url('./Background.jpg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.Start-button {
+  text-align: center
+}
+
+.button {
```

Comment:
Is there a difference between lowercase class names and started from capital letter class names?

Reply:
There is no difference at all. Changed lowercase class names to looklike as others for better readability.